style.css

```diff
@@ -0,0 +1,43 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  background-color: #3c3a39;
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+}
```

Comment:
- [ ] Please, make sure you are using the right font family through out your site. It is important that as developers, we follow the given design closely